src/views/index.js

```diff
@@ -2,5 +2,5 @@
 window.onload = function () {
     const appEl = document.querySelector('#app');
     const isLogged = !!sessionStorage.userLogged;
-    interview.navigate(isLogged ? 'evaluations' : 'login');
+    isLogged ? interview.modules.moduleEvaluations.init() : interview.modules.moduleLogin.init();
```

Comment:
Ternary in lieu of more verbose `if` / `else` block - nice touch! :wink:

Reply:
I knew it was a great idea... You gave me the idea :D